php/php-k8s-v7/etc/nginx/apps/drupal/drupal.conf

```diff
@@ -142,6 +142,32 @@ location / {
         try_files $uri $uri/ @drupal;
     }
 
+    ## Location for public derivative images to avoid hitting Drupal for invalid
+    ## image derivative paths or if the source image doesn't exist.
+    location ^~ /sites/default/files/styles/ {
```

Comment:
Sadly HR.info uses `sites/www.humanitarianresponse.info/files` :-(

Reply:
We could change the regex to capture the site part. Do you think it's worthy?

Reply:
Nah, let's not bother. All future sites will use the default and for HRinfo and unocha.org it only means they simply keep working as they were before.